gnssanalysis/gn_io/igslog.py

```diff
@@ -109,21 +134,25 @@ def parse_igs_log(filename_array: _np.ndarray) -> _np.ndarray:
     with open(file_path, "rb") as file:
         data = file.read()
 
-    blk_id = _REGEX_ID.search(data)
+    blk_id = _REGEX_ID_V2.search(data)
```

Comment:
an unit test can be good so we can have a quick check if it works or not?

Reply:
I've now written up unit tests for all the regex functions in `parse_igs_log()` - which includes the two new ones I've introduced.
I was considering whether to just do a unit test for the entire `parse_igs_log()` function, but I've decided to break it up.
As always, happy to try other suggestions